src/ai/backend/gateway/stream.py

```diff
@@ -322,6 +322,8 @@
     t.Dict({
         tx.AliasedKey(['app', 'service']): t.String,
         tx.AliasedKey(['port'], default=None): t.Null | t.Int[1024:65535],
+        tx.AliasedKey(['arguments'], default=None): t.Null | t.String, # {'-P': '12345'}
```

Comment:
Please include more formal definition of the value format like:
```suggestion
        tx.AliasedKey(['arguments'], default=None): t.Null | t.String,  # stringified JSON, e.g., '{"-P": "12345"}'
                                                                        # For flag-style arguments, the value must be None.
```